README.md

```diff
@@ -34,27 +45,17 @@ Put that URL (only the part within the brackets) into the `.inc` file, saving th
 
 Please have a look at our [style and tone guide](https://jamulus.io/contribute/Style-and-Tone).
 
-## Translating
-
-To make updates or translations for jamulus.io you will need a GitHub account and be familiar with GitHub in general. If you want to get more involved, you’ll need to be familiar with the use of git on the command line, as well as Jekyll. Note that translating the Jamulus application itself is done by a different process, but you are welcome to help with that too. Introduce yourself on the [GitHub Discussion](https://github.com/jamulussoftware/jamulus/discussions) and say what you’d like to do.
-
-### Please also note that discussions should be in English so as to be open to all.
-
-**For details on the website translation process, please consult the [README file here](https://github.com/jamulussoftware/jamuluswebsite/tree/release/_translator-files)**
-
-For translating text in the Jamulus application itself, please see [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/docs/TRANSLATING.md) in the application repository.
-
 ### Overall workflow
 
 We collect changes to the English version of the site on a "next-release" branch first. We then freeze changes prior to a Jamulus software release, and do a translation "sprint" over a couple of weeks when all translation takes place.
 
 1. Changes are first made to EN (= English) *.md files and committed to the “next-release” branch.
 1. Once we’ve agreed the changes can go live (usually just before a software release), we then create GitHub issues for each language, tagged for that release. You can ask questions about the work there.
-1. Translators for each language then update any affected files by editing the .po files for their language in `_translator-files/po/LANGUAGE/` and open pull requests to merge them into the "next-release” branch. Those pull requests are linked to the relevant GitHub issue in step 2 so they can be tracked.
+1. Translators for each language then update them on Weblate, or edit the .po files for their language in `_translator-files/po/LANGUAGE/` and open pull requests to merge them into the "next-release” branch.
 1. When all translations are merged (issues will then close automatically), we merge that new branch into the `release` branch, which is automatically made live on the production site.
 
 ### Points to note
 
-- Please do not add new or significantly modified content to a non-English .html file and do not edit **any** non-English .md files, as these changes would later be lost. All content changes must be made on the "next-release" branch in English **first** after being [discussed here](https://github.com/jamulussoftware/jamulus/discussions).
+- Please do not add new or significantly modified content to a translation. All content changes must be made on the "next-release" branch in English **first** after being [discussed here](https://github.com/jamulussoftware/jamulus/discussions).
```

Comment:
Trying to get this a bit less technical as those details aren't necessary at this point.
```suggestion
- If you see content that is incorrect and needs amending, do not make that change in your translation, as all translators need to know about the correction.  That means updating the English files **first**, after being [discussed here](https://github.com/jamulussoftware/jamulus/discussions).
```

Reply:
Although that could still be better...

Reply:
Hmm, I think it's more about not straying from, and sticking to, what the source EN says - not necessarily correcting mistakes. How about:

- Translations should stay as close as possible to the original English text. If you see content that is incorrect or unclear, please do not correct or rewrite content in your translation, but rather make this fact known by [discussing it here](https://github.com/jamulussoftware/jamulus/discussions) so it can be corrected in the English files **first** and thus apply to all other translations.